kubeflow/core/prototypes/iap-envoy.jsonnet

```diff
@@ -22,8 +22,6 @@ local envoyImage = import 'param://envoyImage';
 local audiencesParam = import 'param://audiences';
 local audiences = std.split(audiencesParam, ',');
 local disableJwtCheckingParam = import 'param://disableJwtChecking';
-local disableJwtChecking = if disableJwtCheckingParam || disableJwtCheckingParam == "true" then
-  true
-else false;
+local disableJwtChecking = disableJwtCheckingParam == "true";
```

Comment:
Can you patch core/util.libsonnet from here
https://github.com/kubeflow/kubeflow/pull/175/files#diff-fbd3d99678fdfe2d482cc3fc2152b615

and then use the function toBool to convert disableJwtCheckingParam?

I think that's better because we want to allow for disableJwtCheckingParam to be a string or a bool